atsign-foundation/at_client_sdk
# latest commit id - handle no response or exception from server

**- What I did**
- when server returns exception or no response, throw exception
**- How I did it**
- added Exception block to SyncUtil.getLatestServerCommitId. rethrow AtClientException
**- How to verify it**
- run sync from at client when server is not reachable
- client should run gracefully by calling the sync error call back.
